bin/test-lxd-network-ovn

```diff
@@ -1014,6 +1014,7 @@ ovn_peering_tests() {
     lxc exec ovn1 -T -n --project=ovn1 -- ping -c1 -6 -W5 "${ovn2NICIPv6}"
     lxc exec ovn2 -T -n --project=ovn2 -- ping -c1 -4 -W5 "${ovn1NICIPv4}"
     lxc exec ovn2 -T -n --project=ovn2 -- ping -c1 -6 -W5 "${ovn1NICIPv6}"
+    lxc network acl show-log ovn1 --project=ovn1
```

Comment:
@tomponline I initially was going to grep for something in the output of this but there doesn't seem to be any output? Is that right?

Reply:
There should only be output if one of the rules that have logging enabled is matched.

Reply:
Right, you'll need to generate a bit of traffic to make that work.

Reply:
Ah I missed the `state=logged` configuration.

Reply:
I've set the state of one of the ACL rules to logged and checked that all the fields are the expected values with jq.